.github/workflows/pr-build-live-branch.yml

```diff
@@ -81,6 +85,14 @@ jobs:
                   # Only need to retain for a day since the beta builder slurps it up to distribute.
                   retention-days: 1
 
+            - name: Create playground artifact
+              uses: actions/upload-artifact@v4
+              if: steps.prepare.outputs.plugin-data != '{}'
+              with:
+                  name: plugins-${{ github.run_id }}
+                  path: unzips/woocommerce
+                  retention-days: 7
```

Comment:
Should we note somewhere in the comment below that the retention period exists, or is there something on the plugin proxy below that keeps the package available beyond the retention period?

Reply:
@jonathansadowski yes that's a good point. Should we increase retention too, maybe to 30 days or something? 7 days is probably too short, I think I was messing around with it originally because I had hoped we could use live branches assets but they don't supply cors headers.

Reply:
set to 30 days here https://github.com/woocommerce/woocommerce/pull/46761/commits/814f47c2aa42603b06fa9f08327093e099f6136c